test/umzug.test.ts

```diff
@@ -399,6 +399,10 @@ describe('types', () => {
 		});
 	});
 
+	test('rerun behavior is a map of its keys to themselves', () => {
+		expectTypeOf(RerunBehavior).toEqualTypeOf<{ readonly [K in RerunBehavior]: K }>();
+	});
```

Comment:
Awesome, you've read my mind. I almost mentioned this in my last review, but I ended up thinking it would be too much hassle for you :sweat_smile: but you did it, that's even cooler.